growth/too/tasks/scheduler.py

```diff
@@ -225,3 +232,81 @@ def get_decam_dict(data_row, queue_name, cnt, nrows,
+
+    t = Table(rows=json_data['targets'])
+    coords = SkyCoord(ra=t['ra'], dec=t['dec'], unit=(u.degree, u.degree))
+    hanle = EarthLocation(lat=32.77889*u.degree,
+                          lon=78.96472*u.degree,
+                          height=4500*u.m)
+    iao = Observer(location=hanle, name="GIT", timezone="Asia/Kolkata")
+
+    twilight_prime = iao.sun_rise_time(Time.now(),
+                                       which="next",
+                                       horizon=sunrise_horizon*u.degree)\
+        - 12*u.hour
+    targets_rise_time = iao.target_rise_time(twilight_prime,
+                                             coords,
+                                             which="nearest",
+                                             horizon=horizon*u.degree)
+    targets_set_time = iao.target_set_time(targets_rise_time, coords,
+                                           which="next",
+                                           horizon=horizon*u.degree)
+    rise_time_IST = (targets_rise_time + 5.5*u.hour).isot
+    set_time_IST = (targets_set_time + 5.5*u.hour).isot
+    tend = targets_set_time
+    mooncoords = get_moon(tend, hanle)
+    sep = mooncoords.separation(coords)
+
+    t.add_column(Column(name='x', length=len(t), dtype='S10'))
+    t.add_column(Column(name='u', length=len(t), dtype='S10'))
+    t.add_column(Column(name='g', length=len(t), dtype='S10'))
+    t.add_column(Column(name='r', length=len(t), dtype='S10'))
+    t.add_column(Column(name='i', length=len(t), dtype='S10'))
+    t.add_column(Column(name='z', length=len(t), dtype='S10'))
+    priority = np.zeros(len(t)) + priority
+    t.add_column(Column(name='Priority', data=priority))
+    t.add_column(Column(name='Dec', data=t['dec']))
+    t.add_column(Column(name='rise_time_IST', data=rise_time_IST))
+    t.add_column(Column(name='set_time_IST', data=set_time_IST))
+    t.add_column(Column(name='moon_angle', data=sep))
+    t.add_column(Column(name='RA', length=len(t), dtype='S12'))
+    t.add_column(Column(name='Target', length=len(t), dtype='S20'))
+    t.remove_column('dec')
+    t.remove_column('request_id')
+    t.remove_column('program_id')
+    t.remove_column('program_pi')
+    t.remove_column('subprogram_name')
+    t.rename_column('field_id', 'tile_id')
+    t.add_column(Column(name='dec', length=len(t), dtype='S20'))
+    domesleeparr = np.zeros(len(t)) + domesleep
+    t.add_column(Column(name='domesleep', data=domesleeparr))
+
+    minalt = AltitudeConstraint(min=horizon*u.degree)
+    always_up = is_always_observable(minalt, iao, coords,
+                                     Time(twilight_prime,
+                                          twilight_prime+12*u.hour))
+
+    bands = {1: 'g', 2: 'r', 3: 'i', 4: 'z', 5: 'J'}
+    for i in range(len(t)):
+        filt = bands[t[i]['filter_id']]
+        t[i]['Target'] = json_data["queue_name"]
+        t[i]['RA'] = '%i:%i:%.2f' % (coords[i].ra.hms[0],
```

Comment:
You can use https://docs.astropy.org/en/stable/coordinates/formatting.html to do this more concisely.